tools/picard-markDuplicates.cwl

```diff
@@ -68,6 +68,7 @@ inputs:
 
   - id: "#removeDuplicates"
     type: ["null", string]
+    default: false
```

Comment:
Do not add default: false if it is tool's default - you can put it in description.